src/pytest_docker/__init__.py

```diff
@@ -18,3 +18,16 @@
     "docker_cleanup",
     "docker_services",
 ]
+
+import pytest
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    group = parser.getgroup("docker")
+    group.addoption(
+        "--container-scope",
+        type=str,
+        action="store",
+        default="module",
```

Comment:
I don't think it is okay to change the default scope for fixtures. It should be "session"

Reply:
```suggestion
        default="session",
```